### PR TITLE
[DDW-579] Undelegated wallet shown as delegated on "Delegation center" screen

### PR DESCRIPTION
**Pull request by yakovkaravelov**
This PR fixed delegated stake pool status on Delegation Center screen
[Jira Ticket](https://jira.iohk.io/browse/DDW-579)

## Todos
- [x] Fix delegated stake pool status on Delegation Center screen

## Testing Checklist
- [Slack QA thread](https://input-output-rnd.slack.com/archives/GGKFXSKC6/p1614262917027700)

## Test Cases

```Gherkin
Scenario 1: Wallet undelegated on Yoroi shown as delegated on Daedalus
Given I have undelegated a wallet on Yoroi
When I go to Daedalus Delegation Center
Then this wallet will be shown as undelegated
```

Scenario: Wallet undelegated 

### App
- [x] Go to `Staking` => `Delegation Center` and make sure delegated stake pool statuses of wallets are properly shown up

### English
image

### Japanese
image

### Basics

- [ ] PR has been assigned and has appropriate labels (`feature`/`bug`/`chore`, `release-x.x.x`)
- [ ] PR is updated to the most recent version of the target branch (and there are no conflicts)
- [ ] PR has a good description that summarizes all changes and shows some screenshots or animated GIFs of important UI changes
- [ ] CHANGELOG entry has been added to the top of the appropriate section (*Features*, *Fixes*, *Chores*) and is linked to the correct PR on GitHub
- [ ] Automated tests: All acceptance and unit tests are passing (`yarn test`)
- [ ] Manual tests (minimum tests should cover newly added feature/fix): App works correctly in *development* build (`yarn dev`)
- [ ] Manual tests (minimum tests should cover newly added feature/fix): App works correctly in *production* build (`yarn package` / CI builds)
- [ ] There are no *flow* errors or warnings (`yarn flow:test`)
- [ ] There are no *lint* errors or warnings (`yarn lint`)
- [ ] There are no *prettier* errors or warnings (`yarn prettier:check`)
- [ ] There are no missing translations (running `yarn manage:translations` produces no changes)
- [ ] Text changes are proofread and approved (Jane Wild / Amy Reeve)
- [ ] Japanese text changes are proofread and approved (Junko Oda)
- [ ] UI changes look good in all themes (Alexander Rukin)
- [ ] Storybook works and no stories are broken (`yarn storybook`)
- [ ] In case of dependency changes `yarn.lock` file is updated

### Code Quality
- [ ] Important parts of the code are properly commented and documented
- [ ] Code is properly typed with flow
- [ ] React components are split-up enough to avoid unnecessary re-renderings
- [ ] Any code that only works in main process is neatly separated from components

### Testing
- [ ] New feature/change is covered by acceptance tests
- [ ] New feature/change is manually tested and approved by QA team
- [ ] All existing acceptance tests are still up-to-date
- [ ] New feature/change is covered by Daedalus Testing scenario
- [ ] All existing Daedalus Testing scenarios are still up-to-date

### After Review
- [ ] Merge the PR
- [ ] Delete the source branch
- [ ] Move the ticket to `done` column on the YouTrack board
- [ ] Update Slack QA thread by marking it with a green checkmark

### CODE REVIEW

**Comment by nikolaglumac** [comment]
@yakovkaravelov please merge latest develop.

File: source/renderer/app/components/staking/delegation-center/WalletRow.js [hunk 1]
```diff
@@ -166,6 +167,48 @@ export default class WalletRow extends Component<Props, WalletRowState> {
...
+      (delegation: WalletNextDelegation) =>
+        get(delegation, ['changes_at', 'epoch_number'], 0) === epochNumber
+    );
+
+    if (!foundDelegation) {
+      return fallbackStakePoolId;
+    }
+
+    if (
+      get(foundDelegation, 'status') === WalletDelegationStatuses.NOT_DELEGATING
```
Comment (tomislavhoracek) [inline source/renderer/app/components/staking/delegation-center/WalletRow.js:204]: @yakovkaravelov isolate this to e.g. `const isDelegating`

**Review by tomislavhoracek — CHANGES_REQUESTED** [review]
@yakovkaravelov Nice job.
I left just one comment. 
P.S. Why this is changed if you didn't touch translations? Can you check this please
`source/renderer/app/i18n/locales/defaultMessages.json`
And also merge latest `develop` branch and fix conflicts

**Comment by yakovkaravelov** [comment]
> @yakovkaravelov Nice job.
> I left just one comment.
> P.S. Why this is changed if you didn't touch translations? Can you check this please
> `source/renderer/app/i18n/locales/defaultMessages.json`
> And also merge latest `develop` branch and fix conflicts

@tomislavhoracek `defaultMessage.json` can be changed when there is line number change regarding translation definitions even if there is no change in translation resources.
It happens after running `yarn manage:translations`

**Comment by miorsufianiohk** [comment]
@yakovkaravelov  Unable to test as not able to undelegate on Yoroi due to transaction issue on testnet. See [screenshot](https://drive.google.com/file/d/1yUmGZjQYmhDEkgLLIPRBGsJmmlHPRk1u/view?usp=sharing)

**Review by tomislavhoracek — APPROVED** [review]
@yakovkaravelov code looks good now 👍

**Comment by darko-mijic** [comment]
Reviewing this now with Trezor HW wallet on the mainnet...

**Comment by darko-mijic** [comment]
image
Works great. IOHK Trezor 2 is now undelegated.

**Review by darko-mijic — APPROVED** [review]

**Review by gabriela-ponce — APPROVED** [review]
I undelegated in Yoroi and the changes are visible in Daedalus for [Ledger S](https://drive.google.com/file/d/1InJt5NhW5dGHR4ZwVgnIm6EPblDpODx8/view?usp=sharing) and [software wallets](https://drive.google.com/file/d/1nimqHZypmd2x8JdCp5pvuUKt6uu7kgFp/view?usp=sharing).

**Review by miorsufianiohk — APPROVED** [review]
Tested with software wallet on Yoroi testnet and Daedalus testnet. Good work @yakovkaravelov 👍

**Review by nikolaglumac — APPROVED** [review]

**Pull request merged**